config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -216,6 +220,15 @@ elseif (BOARD STREQUAL "native_posix_64")
     chip_gn_arg_string("target_cpu" "x64")
 endif()
 
+if (CONFIG_CHIP_PW_RPC)
```

Comment:
Actually, I was planning to remove all GN code for Pigweed on nRF Connect platform since Pigweed is currently included using CMake directly (see lighting-app/nrfconnect/CMakeLists.txt). Can the tracing be enabled in CMake?

Reply:
These dependencies are actually in lighting-app/nrfconnect/CMakeLists.txt. There are used to build the main app with rpc turned on and the dependencies are used for building RPC.cpp. However the dependencies are not passed down to chip gn to build chip core library. We have pigweed dependencies in CASE and PASE (where trace events are inserted for analysis). That is the reason why these dependencies are added in chip-module/CMakeLists.txt